### PR TITLE
Refactor the submit queue manifests

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: cluster/ci/config/submit-queue/submit_queue.yaml [hunk 1]
```diff
@@ -1,133 +1,23 @@
 apiVersion: v1
-kind: Template
```
Comment (kargakis) [inline cluster/ci/config/submit-queue/submit_queue.yaml]: Do you also need to update make targets because of this change?

**Comment by kargakis** [comment]
lgtm minus a comment

File: cluster/ci/config/submit-queue/submit_queue.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply cluster/ci/config/submit-queue/submit_queue.yaml]: yarp

**Comment by stevekuznetsov** [comment]
@kargakis updated the `Makefile`

**Pull request merged**